site/app/libraries/socket/Client.php

```diff
@@ -16,7 +16,12 @@ public function __construct(Core $core) {
         ]);
     }
 
-    public function send($payload, $opcode = 'text', $masked = true): void {
```

Comment:
Overriding the `send` function was causing an error in my client where `$payload` should only ever be a string. Not sure why phpstan wasn't flagging this as an error in the build. Given the likelihood the parent type may become more strongly typed in the future and this becomes a proper error, better to just avoid it, and add a new function that expects to `json_encode` its payload before calling `send`.